examples/fcrepo-camel-osgi/src/main/resources/OSGI-INF/blueprint/camel-context.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0"
+       xmlns:camel="http://camel.apache.org/schema/blueprint"
+       xsi:schemaLocation="
+       http://www.osgi.org/xmlns/blueprint/v1.0.0 http://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd
+       http://camel.apache.org/schema/blueprint http://camel.apache.org/schema/blueprint/camel-blueprint.xsd">
+
+   <!-- OSGI blueprint property placeholder -->
+   <cm:property-placeholder persistent-id="org.fcrepo.camel.examples.osgi" update-strategy="reload">
+     <cm:default-properties>
+       <cm:property name="fcrepo.baseUrl" value="localhost:8080/rest"/>
+       <cm:property name="fcrepo.defaultTransform" value="default"/>
+       <cm:property name="fcrepo.authUsername" value=""/>
+       <cm:property name="fcrepo.authPassword" value=""/>
+       <cm:property name="fcrepo.secure" value="false"/>
+       <cm:property name="jms.brokerUrl" value="tcp://localhost:61616"/>
+       <cm:property name="jms.fcrepoEndpoint" value="topic:fedora"/>
+       <cm:property name="triplestore.baseUrl" value="localhost:3030/ds"/>
+       <cm:property name="error.dir" value="/tmp/camel"/>
+       <cm:property name="error.file" value="errors.log"/>
+       <cm:property name="rest.port" value="9080"/>
+       <cm:property name="rest.prefix" value="/index"/>
+     </cm:default-properties>
+   </cm:property-placeholder>
+
+  <!-- configuration of activemq component -->
+  <bean id="activemq" class="org.apache.activemq.camel.component.ActiveMQComponent">
+    <property name="brokerURL" value="${jms.brokerUrl}"/>
+  </bean>
+
+  <!-- component-wide configuration of fcrepo -->
+  <bean id="fcrepo" class="org.fcrepo.camel.FcrepoComponent">
+    <property name="authUsername" value="${fcrepo.authUsername}"/>
+    <property name="authPassword" value="${fcrepo.authPassword}"/>
+    <property name="authHost" value="${fcrepo.authHost}"/>
+    <property name="secure" value="${fcrepo.secure}"/>
+  </bean>
+
+  <camelContext xmlns="http://camel.apache.org/schema/blueprint">
+    <package>org.fcrepo.camel.examples.osgi</package>
+
+    <!-- Configuration of REST indexing endpoint -->
+    <restConfiguration component="restlet" port="{{rest.port}}"/>
```

Comment:
Yes, the {{...}} syntax is for property values. The difference is that you can use {{...}} inside a camel context (i.e. in the URIs) while ${...} can only be used outside the camel context (i.e. in a bean definition) -- the property values, however, can all be defined in a single configuration file. The documentation for this is here: http://camel.apache.org/using-propertyplaceholder.html